src/GraphQL/AssetType/AssetType.php

```diff
@@ -67,55 +61,61 @@ public function build(&$config)
         $propertyType = $this->getGraphQlService()->buildGeneralType('element_property');
         $elementResolver = new Resolver\Element('asset', $this->getGraphQlService());
 
+        $resolutionsType = Type::listOf(new ObjectType([
+            'name' => 'resolutions',
+            'fields' => [
+                'url' => Type::string(),
+                'resolution' => Type::int(),
+            ],
+        ]));
+        $resolutionsArgumentsType = [
+            'type' => Type::listOf(Type::int()),
```

Comment:
I believe this must change as well @weisswurstkanone :)

Reply:
:-)